install_files/ansible-base/callback_plugins/ansible_version_check.py

```diff
@@ -19,7 +19,7 @@ def print_red_bold(text):
 class CallbackModule(CallbackBase):
     def __init__(self):
         # Can't use `on_X` because this isn't forwards compatible with Ansible 2.0+
-        required_version = '2.3.2'  # Keep synchronized with group_vars/all/main.yml
+        required_version = '2.4.2'  # Keep synchronized with requirements files
```

Comment:
I edited this postfix comment because it didn't seem accurate. If someone disagrees, happy to discuss.